### PR TITLE
Refactor project generation so that contract is copied from single so…

### PR DESCRIPTION
**Pull request by janedegtiareva**
…urce

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -59,6 +59,17 @@ const doCreateProject = async function(options) {
         to: projectName
     });
 
+    // copy contract
+    const contractTargetDir = `${projectDir}/${options.rust? 'contract' : 'assembly'}`;
+    const contractSourceDir = `${__dirname}/contract/${options.rust? 'rust' : 'asc'}`;
+    console.log(`Copying contract files to new project directory (${contractTargetDir}) from source (${contractSourceDir}).`);
+    const copyContractDirFn = () => {
+        return new Promise(resolve => {
+            ncp(contractSourceDir, contractTargetDir, response => resolve(response));
```
Comment (vgrichina) [inline index.js]: This doesn’t propagate errors to caller. Use util.promisify?

**Comment by icerove** [comment]
could you let the ci pass first?

**Comment by janedegtiareva** [comment]
> Lgtm overall. Not sure about:
> 
> * shouldn’t there be change to index.ja in web template copying? E. g. To avoid it erroring on rust + react
> * looks like blank rust project folder still remains because of gitkeep file.

the change to generate rust + react is in flight.
We can't quite get rid of rust project yet because of readme file and package.json diffs.

File: index.js (hunk 1 above)
Comment (janedegtiareva) [reply index.js]: Filed a reminder issue to track, will address later when rust + react are done

**Pull request merged**